Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -7447,6 +7447,8 @@ void WebPageProxy::createNewPage(WindowFeatures&& windowFeatures, NavigationActi
 
     Ref configuration = this->configuration().copy();
     configuration->setRelatedPage(*this);
+    configuration->setBrowsingContextGroup(m_browsingContextGroup);
+    configuration->setOpenerFrameID(originatingFrameInfoData.frameID);
```

Comment:
Does `configuration().copy()` not copy `openerFrameID` and `browsingContextGroup`?

Reply:
It does, but the openerFrameID was nullopt and now we need it to be set.  So the first of these two lines isn't needed but the second is.